yt/odin/checks/bin/unaware_nodes/__main__.py

```diff
@@ -1,9 +1,26 @@
 from yt_odin_checks.lib.check_runner import main
 
+DEFAULT_ENABLED_NODE_FLAVORS = [
```

Comment:
Don't you want to make default list empty (or none) and not check flavors in this case? New node flavors may appear from time to time and adding them here is a little bit counterintuitive.

Reply:
yeah, it makes sense